packages/core/src/payment/strategies/stripev3/stripev3.ts

```diff
@@ -549,6 +549,11 @@ export interface StripeElement {
      * Call `element.mount` to re-attach it to the DOM.
      */
     unmount(): void;
+
+    /**
+     * Listend changes
```

Comment:
https://stripe.com/docs/js/element/events/on_change?type=paymentElement

Reply:
Thanks @mauricio-sg, also I created an interface with the change only for stripeUpe